### PR TITLE
Convert component import/exports to ES6

### PR DESCRIPTION
**Pull request by geemanjs**

### CODE REVIEW

File: src/types/index.js [hunk 1]
```diff
@@ -2,4 +2,4 @@ module.exports = {
   value: require('./value'),
```
Comment (lennym) [inline src/types/index.js]: This one got left behind. I don't know if that was intentional.

**Comment by lennym** [comment]
This will need the build task modifying so that it compiles away the `imports`.

File: src/types/index.js (hunk 1 above)
Comment (geemanjs) [reply src/types/index.js]: Thanks not intentional

**Comment by lennym** [comment]
Looks great. Thanks a lot.

**Pull request merged**